src/python/zquantum/core/interfaces/estimator.py

```diff
@@ -4,15 +4,18 @@
 from ..measurement import ExpectationValues
 from abc import ABC, abstractmethod
 from openfermion import SymbolicOperator
-from typing import List, Optional
+from typing import Optional
+from overrides import EnforceOverrides
 
+logger = logging.getLogger(__name__)
 
-class Estimator(ABC):
+
+class Estimator(ABC, EnforceOverrides):
```

Comment:
I might be missing something, but can you clarify how is adding `EnforceOverrides` related to the changes to the logger placement proposed here? Or is it just an unrelated change?

Reply:
Not missing anything! It's an unrelated change. I updated the PR body.